gcn/notices/svom/eclairs/eclairs.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/eclairs/eclairs.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM ECLAIRs alerts",
+  "description": "ECLAIRs detection information of a GRB or other X-Ray transient",
+  "allOf": [
+    { "$ref": "../../core/Alert.schema.json" },
+    { "$ref": "../../core/Reporter.schema.json" },
+    { "$ref": "../../core/Event.schema.json" },
+    { "$ref": "../../core/DateTime.schema.json" },
+    { "$ref": "../../core/Localization.schema.json" },
+    { "$ref": "../../core/Statistics.schema.json" },
+    { "$ref": "../../core/Pointing.schema.json" },
+    { "$ref": "../../core/GeoLocBase.schema.json" },
+    { "$ref": "../../core/AdditionalInfo.schema.json" }
+  ],
+  "properties": {
+    "$schema": true,
+    "slew_status": {
+      "enum": ["accepted", "rejected", "not_requested"],
+      "description": "Whether the platform accepted or rejected a slew requested by ECLAIRs (to its best localisation). If the detection is not significant enough, this value can remain to 'not_requested'"
```

Comment:
```suggestion
      "description": "Whether the platform accepted or rejected a slew requested by ECLAIRs (to its best localisation). If the detection is not significant enough, this value is 'not_requested'."
```

Reply:
Is there a simple prescription for the significance or a document you can reference where it's described?